datacube/api/core.py

```diff
@@ -447,7 +447,7 @@ def empty_func(measurement_):
         for name, coord in coords.items():
             result[name] = coord
         for name, coord in geobox.coordinates.items():
-            result[name] = (name, coord.values, {'units': coord.units})
+            result[name] = (name, coord.values, {'units': coord.units, 'resolution': coord.resolution})
```

Comment:
@uchchwhash why do we need resolution field? It is trivially computed from coordinate values, who will ensure consistency of this field with actual data?